uint8 cmd
bool isUrgent
int16[4] data